### PR TITLE
fix: alphabetically sort remote apps

### LINKED ISSUE #1713: Recommended apps on NFT section are not in order
## Bug description
Recommended apps on NFT section get a random order

## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Go to NFT assets page
 2. Suggested apps are not alphabetically ordered
 3. Castle did an extra effort integrating and should be presented on a higher position

## Expected result
The old alphabetical order behavior is keeped.

## Obtained result
Unordered app list

## Screenshots
imagen

## Links
https://github.com/5afe/safe-apps-list/issues/223

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Resolves #1713

## How this PR fixes it

`useRemoteSafeApps` sorts the remote apps alphabetically.

## How to test it

1. Open the Safe Apps list and observe it alphabetically sorted (before filter).
2. Open the NFT list and observe the NFT Safe Apps sorted alphabetically.

## Screenshots

image

image

## Checklist
* [x] I've tested the branch on mobile 📱
* [x] I've documented how it affects the analytics (if at all) 📊
* [x] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/hooks/safe-apps/useRemoteSafeApps.ts [hunk 1]
```diff
@@ -41,10 +41,18 @@ const useRemoteSafeApps = (tag?: SafeAppsTag): AsyncResult<SafeAppsResponse> =>
 
   const apps = useMemo(() => {
     if (!remoteApps || !tag) return remoteApps
-    return remoteApps.filter((app) => app.tags.includes(tag))
+    return remoteApps
+      .filter((app) => app.tags.includes(tag))
+      .sort((a, b) => {
+        return a.name < b.name ? -1 : a.name > b.name ? 1 : 0
+      })
```
Comment (katspaugh) [inline src/hooks/safe-apps/useRemoteSafeApps.ts:48]: Can we avoid code repetition here?
```
const apps = tag ? apps.filter(tag) : apps
return apps.sort()
```

Comment (iamacook) [reply src/hooks/safe-apps/useRemoteSafeApps.ts]: Good catch! I created `sortedApps` to remove this but forgot to do so 🙈

**Review by katspaugh — APPROVED** [review]
👍

**Comment by katspaugh** [comment]
I've tested it and confirm that the apps are now sorted, both in the main list and on the NFT page. ✅

**Pull request merged**